scripts/cli/tests/13_confidential_assets.js

```diff
@@ -0,0 +1,75 @@
+// Set options as a parameter, environment variable, or rc file.
+require = require("esm")(module /*, options*/);
+module.exports = require("../util/init.js");
+
+let { reqImports } = require("../util/init.js");
+// Sets the default exit code to fail unless the script runs successfully
+process.exitCode = 1;
+
+const prepend = "EUR";
+const prepend2 = "USD";
+
+
+async function main() {
+  const api = await reqImports.createApi();
+  const tickerHex = reqImports.stringToHex(`${prepend}0`); 
+  const ticker2Hex = reqImports.stringToHex(`${prepend2}0`); 
+  const testEntities = await reqImports.initMain(api);
+
+  let alice = testEntities[0];
+  let bob = testEntities[1];
+  let charlie = testEntities[2];
+
+
+  let alice_did = await reqImports.keyToIdentityIds(api, alice.publicKey);
+
+  let bob_did = await reqImports.createIdentities(api, [bob], alice);
+  bob_did = bob_did[0];
+
+  let charlie_did = await reqImports.createIdentities(api, [charlie], alice);
+  charlie_did = charlie_did[0];
```

Comment:
Would you be able to create these two identities with a single call?
```suggestion
  let dids = await reqImports.createIdentities(api, [bob, charlie], alice);
  let bob_did = dids[0];
  let charlie_did = dids[1];
```

Reply:
Yes, I would be optimizing the code later.